lib/Module/CMakeLists.txt

```diff
@@ -23,9 +23,14 @@ set(LLVM_COMPONENTS
   bitreader
   bitwriter
   codegen
+  instcombine
```

Comment:
So the difference is:
```
llvm_map_components_to_libraries(LLVM_LIBS1 instcombine)
llvm_map_components_to_libnames(LLVM_LIBS2 instcombine)

message("old ${LLVM_LIBS1}")
message("new ${LLVM_LIBS2}")
```
```
old LLVMInstCombine;LLVMTransformUtils;LLVMAnalysis;LLVMProfileData;LLVMObject;LLVMMCParser;LLVMMC;LLVMBitReader;LLVMCore;LLVMSupport;LLVMDemangle
new LLVMInstCombine
```

So we should use llvm_map_components_to_libnames differently, apparently :).